packages/theme-chalk/src/button.scss

```diff
@@ -232,6 +232,19 @@
       z-index: 1;
     }
   }
+  
+  & > .el-dropdown {
+    & > .el-button {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+      
+      @each $type in (primary, success, warning, danger, info) {
```

Comment:
Is it necessary to use a loop?
```css
border-left-color: rgba($--color-white, 0.5);
```
This also should work.

Reply:
When Dropdown is the last child in ButtonGroup, the Button which in Dropdown isn't be considered the last child in the ButtonGroup, and the Dropdown is usually used as the last child in the ButtonGroup, so I add below loop style.
```css
@each $type in (primary, success, warning, danger, info) {
  &.el-button--#{$type} {
    border-left-color: rgba($--color-white, 0.5);
  }
}
```
![image](https://user-images.githubusercontent.com/7844486/45988034-41c06780-c0a7-11e8-908d-fe7c2f606419.png)

Reply:
@ziyoung Sorry, I misunderstood your point.I had modified style source.Pls confirm it.